cockroachdb/datadog_checks/cockroachdb/data/conf.yaml.example

```diff
@@ -5,6 +5,21 @@ instances:
   ## The Prometheus endpoint to query.
   #
   - prometheus_url: http://localhost:8080/_status/vars
+  
+  ## @param ssl_cert - string - optional 
+  ## If the certificate contains the ssl_private_key you can use this optional param instead of ssl_private_key and ssl_ca_cert
+  #
+  # ssl_cert: <path_to_cert.pem>
+
+  ## @param ssl_private_key - string - optional
+  ## Needed if the certificate does not include the private key. Currently, Requests does not support using encrypted keys.
+  #
+   ssl_private_key: <path_to_cert.key>
+
+  ## @param ssl_ca_cert - string - optional
+  ## The path to the trusted CA used for generating custom certificates
+  #
+   ssl_ca_cert: <path_to_cert.pem>
```

Comment:
If the parameter is optional then it should be commented by default and the Placeholder variable should be capitalized to follow the contributing guidelines. `<PATH_TO_CA_CERT_PEM>`